modules/consensus/difficulty_test.go

```diff
@@ -285,10 +285,13 @@ func TestStoreBlockTotals(t *testing.T) {
 	err = cs.db.Update(func(tx *bolt.Tx) error {
 		totalTime := int64(0)
 		totalTarget := types.RootDepth
-		var id types.BlockID
-		parentTimestamp := types.Timestamp(0)
-		currentTimestamp := types.Timestamp(0)
-		currentTarget := types.RootTarget
+		var (
+			id              types.BlockID
+			parentTimestamp types.Timestamp
+
+			currentTimestamp = types.Timestamp(0)
+			currentTarget    = types.RootTarget
+		)
```

Comment:
this is an odd replacement. Looks like maybe it's complaining about `parentTimestamp` being assigned a value, but then that value is never used?

We don't use `var` blocks much, so we should replace this with:
```go
var id types.BlockID
var parentTimestamp, currentTimestamp types.Timestamp
currentTarget := types.RootTarget
```

Reply:
yeah that could work as well, what's the reason for not using var blocks? To keep diffs as small as possible?